src/static/js/timeslider.js

```diff
@@ -28,11 +30,81 @@ const Cookies = require('./pad_utils').Cookies;
 const randomString = require('./pad_utils').randomString;
 const hooks = require('./pluginfw/hooks');
 
-let token, padId, export_links;
+let token, padId, exportLinks, socket, changesetLoader, BroadcastSlider;
+const fireWhenAllScriptsAreLoaded = [];
```

Comment:
Moving the code around makes it difficult to review changes. Would you move these back to the order they were originally in? (That might not be possible if the original order throws `ReferenceError`.) ESLint will complain about `no-use-before-define`, but those are warnings not errors.

Reply:
afaik this was to fix an error (probably not ``initialized``), I didn't address lint warnings..